product/versioning.md

```diff
@@ -0,0 +1,24 @@
+# Versioning
+
+## 1.0
+
+The 1.0 Release of FlowForge represents a significant milestone, we are making a statement to our customers about the product.
+
+Therefore the following are considerations & requirements in order to be able to stand behind that statement.
+
+- We have the admin features required to manage the platform at an appropriate scale.
```

Comment:
I agree with this list. My doubt and fear is still that this becomes a large chunck of work that's not tightly enough scoped. With that we don't ship fast enough. Hope that makes sense?

I tried to circumvent this with #85. So if we timebox the release with this list, I'm supportive.